students/TrkCat/Lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,10 @@
+for num in range(1:101):
+    str = ('')
```

Comment:
`str` is a built-in python function which you are overwriting

github highlights built-ins for you (so do some IDEs), so make sure to use alternate var names

Reply:
also, the parens are redundant

Reply:
Thank you! Definitely don't want to be in the habit of overwriting built-in functions, I'll update the name.